packages/database/src/migrations/20200924001437-AddSortOrderToMapOverlayGroupRelationTable-modifies-schema.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function(db) {
+  await db.runSql(`
+    ALTER TABLE map_overlay_group_relation 
+    ADD COLUMN sort_order real;
```

Comment:
I know that we currently use `float4` (= `real`) for map overlay sort order, but we really shouldn't: we should use an integer type.

Based on the [Postgres docs on integer types](https://www.postgresql.org/docs/9.6/datatype-numeric.html), I believe we should go for `integer`. It will consume the same amount of memory with a `float4` type (4 bytes), but it represents the data model more accurately (+arithmetic/conversion may be more optimised, although that is not a concern).

Reply:
Thanks Kostas! The reason I kept them as `real` data type is some of the existing sortOrders of our Overlays are in decimal (I'm not sure why)...

However, I'm happy to convert all of them to integers so that we can use integer type in this case!

Reply:
Yeah I'm pretty sure that was for convenience (infinite floats exist between two floats, while finite between two integers...) But this seems a big hack to me 😛 

Like, if you have sort order of 1 and 2, you can sneak in 1.5 there, then 1.6, then 1.55 etc...